scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -53,7 +53,7 @@
     mount:
       state: mounted
       fstype: nfs
-      opts: ro,nosuid,rsize=8192,wsize=8192,hard,intr,nolock
+      opts: ro,nosuid,rsize=8192,wsize=8192,hard,intr,nolock,x-systemd.automount,noauto
```

Comment:
Are we using `systemd-mount`? I was under the impression (and I might be wrong) that we use standard `mount`.

Reply:
Standard mount in the task. Ansible mount module will configure /etc/fstab. The extra added options instruct systemd.mount to treat it as automount after a reboot.(systemd.mount will look over /etc/fstab after a reboot)